### PR TITLE
Added resetlevels command

### PR DESCRIPTION
**Pull request by joshualyon7**

### CODE REVIEW

File: src/commands/admin/resetlevels.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import { Message } from 'discord.js';
+
+export default class extends Command {
+
+	description = 'Resets every user in the guild\'s level to 1';
+
+	enabled = false;
+
+	async permissions(msg: Message): Promise<boolean> {
+		return await botMasterPerms(msg);
```
Comment (BenSegal855) [inline src/commands/admin/resetlevels.ts:13]: Unnecessary `await`. I believe the command handler will await it anyway.

File: src/commands/admin/resetlevels.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		});
+
+		msg.client.mongo.collection(DB.USERS).updateMany({}, { $set: {
+			count: 0,
+			levelExp: 0,
+			level: 0,
+			curExp: FIRST_LEVEL
+		} });
+
+		return await msg.reply('I\'ve reset all levels in the guild.');
```
Comment (BenSegal855) [inline src/commands/admin/resetlevels.ts:40]: Unnecessary `await`.

File: src/commands/admin/resetlevels.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import { DB, FIRST_LEVEL, ROLES } from '@root/config';
+import { botMasterPerms } from '@root/src/lib/permissions';
+import { Command } from '@root/src/lib/types/Command';
```
Comment (BenSegal855) [inline src/commands/admin/resetlevels.ts:3]:
```suggestion
import { botMasterPerms } from '@lib/permissions';
import { Command } from '@lib/types/Command';
```
Make full use of those aliases

**Review by BenSegal855 — COMMENTED** [review]
Overall this code should work with no changes. The comments I left are pretty much all stylistic changes.

**Review by renross — APPROVED** [review]

**Pull request merged**